controller/persistence/test_datastore.py

```diff
@@ -0,0 +1,122 @@
+
+# add parent folder tot system path so python will find persistence package
```

Comment:
Great unit tests,
I think we should place unit tests centralized for each OMA-ML component.
So controller unit tests, should get their dedicated folder 
Controller/tests